airqo-platform/AirQo-api
# Backend: Device Incident Report

- What does this PR do?
This PR updates the production DB details of the data management service
Additionally, it also has 4 endpoints that are to be tested along side the frontend PR for Device Incident Report. These endpoints are the engine for the device incident report.
The PR also deals with security aspects of the security pass for JWT security pass
- What JIRA task is related to this PR?
[Incident Report](https://airqoteam.atlassian.net/browse/NET-6)
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/data-mgt` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
> - api/v1/data/channels/maintenance/add
> - api/v1/data/channels/maintenance
> - api/v1/data/channels/issues/add
> - api/v1/data/channels/issues

 [REST API [data-mgt]](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=757070984)
